EventFilter/Utilities/interface/FedRawDataInputSource.h

```diff
@@ -47,18 +45,18 @@ friend struct InputChunk;
 
 public:
   explicit FedRawDataInputSource(edm::ParameterSet const&,edm::InputSourceDescription const&);
-  virtual ~FedRawDataInputSource();
+  ~FedRawDataInputSource() override;
   static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
 
   std::pair<bool,unsigned int> getEventReport(unsigned int lumi, bool erase);
 protected:
-  virtual bool checkNextEvent() override;
-  virtual void read(edm::EventPrincipal& eventPrincipal) override;
+  bool checkNextEvent() override;
+  void read(edm::EventPrincipal& eventPrincipal) override;
 
 private:
-  virtual void preForkReleaseResources() override;
-  virtual void postForkReacquireResources(std::shared_ptr<edm::multicore::MessageReceiverForSource>) override;
-  virtual void rewind_() override;
+  void preForkReleaseResources() override;
```

Comment:
This method is not there in 94X: is it correct to have it here? Or, alternatively, to have it removed in 94X?